tests/test_exception.py

```
@@ -0,0 +1,39 @@
+fix		           1 all nve
+fix              2 all ave/time 10 1 10 c_1 c_2
+
+run		1000
+"""
+
+
+class TestException(TestCase):
+    def test_overlapping_atoms(self):
+        with open("in.error", "w") as f:
+            f.writelines(lmp_str)
+        with self.assertRaises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)